src/WaveTrack.cpp

```diff
@@ -763,9 +764,19 @@ Track::Holder WaveTrack::Copy(double t0, double t1, bool forClipboard) const
 
          //wxPrintf("copy: clip_t0=%f, clip_t1=%f\n", clip_t0, clip_t1);
 
-         newClip->Offset(-t0);
-         if (newClip->GetPlayStartTime() < 0)
+         // If we are in the case with an audio clip on the right side with blank space to its left,
+         // we need to calculate the offset slightly differently:
+         //  |    [-----CLIP--|--]             [-|----CLIP----]      |
+         // t0                t1       VS.       t0                  t1
+         //  |----|                            |-|
+         //  OFFSET = clip_t0 - t0            OFFSET = 0
+         if (t0 < clip->GetPlayStartTime() && t1 < clip->GetPlayEndTime()) {
```

Comment:
`t1 < clip->GetPlayEndTime()` condition is redundant. Both `t1 >= clip->GetPlayEndTime()` and `t0 < clip->GetPlayStartTime()` isn't possible in that if-branch